storage/src/mmr/proof.rs

```diff
@@ -103,37 +102,37 @@ impl<D: Digest> Default for Proof<D> {
 }
 
 impl<D: Digest> Proof<D> {
-    /// Return true if this proof proves that `element` appears at position `element_pos` within the
-    /// MMR with root digest `root`.
+    /// Return true if this proof proves that `element` appears at location `loc` within the MMR
+    /// with root digest `root`.
     pub fn verify_element_inclusion<I, H>(
         &self,
         hasher: &mut H,
         element: &[u8],
-        element_pos: u64,
+        loc: u64,
```

Comment:
`adb::current::verify_range_proof` passes in a position rather than a location to this function